german/finger-ins-loch/hoelzernes-Paddel/sicherer_Entfernung/gruessen_freundlich/abhauen/abhauen.md

```diff
@@ -0,0 +1,3 @@
+Die beiden Personen scheinen ihre eigenen Probleme zu haben, weshalb sie abhauen.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.